*** call.c	Tue Apr 17 00:20:42 2001
--- call.c.new	Tue Oct 23 16:06:20 2001
***************
*** 38,44 ****
  #include <sys/wait.h>
  #include <termios.h>
  #include <unistd.h>
! #include <curses.h>
  
  #ifdef HAVE_NETAX25_AX25_H
  #include <netax25/ax25.h>
--- 38,44 ----
  #include <sys/wait.h>
  #include <termios.h>
  #include <unistd.h>
! #include <ncurses/curses.h>
  
  #ifdef HAVE_NETAX25_AX25_H
  #include <netax25/ax25.h>
