*** menu.c	Tue Apr 10 03:58:49 2001
--- menu.c.new	Tue Oct 23 16:06:38 2001
***************
*** 6,12 ****
   * 25.07.1995  some minor changes                   Alexander Tietzel (DG6XA)
   */
  
! #include <curses.h>
  #include <stdlib.h>
  #include <string.h>
  #include "menu.h"
--- 6,12 ----
   * 25.07.1995  some minor changes                   Alexander Tietzel (DG6XA)
   */
  
! #include <ncurses/curses.h>
  #include <stdlib.h>
  #include <string.h>
  #include "menu.h"
